gradle/libs.versions.toml

```diff
@@ -69,7 +69,7 @@ truth = { module = "com.google.truth:truth", version = "1.1.5" }
 
 # Plugins
 plugin-android = { module = "com.android.tools.build:gradle", version.ref = "agp" }
-plugin-buildConfig = { module = "com.github.gmazzo:gradle-buildconfig-plugin", version = "3.1.0" }
+plugin-buildConfig = { module = "com.github.gmazzo.buildconfig:plugin", version = "4.1.2" }
```

Comment:
Looks like the registered coordinates, per: https://plugins.gradle.org/plugin/com.github.gmazzo.buildconfig

Reply:
I explicitly used the marker coordinates of `id(com.github.gmazzo.buildconfig)` because that's more stable than artifacts. If it was already the marker, this upgrade would've triggered automatically.